stock_storage_type/models/stock_quant_package.py

```diff
@@ -18,6 +18,15 @@ class StockQuantPackage(models.Model):
         compute="_compute_height_in_m",
         store=True,
     )
+    package_storage_type_id = fields.Many2one(
```

Comment:
@henrybackman @grindtildeath What is the behavior on an existing database ?
IMHO, this compute will last a long time